scripts/apollo_build.sh

```diff
@@ -115,6 +116,13 @@ function _parse_cmdline_arguments() {
         optarg="${!pos}"
         known_options="${known_options} ${opt} ${optarg}"
         ;;
+      --distdir*)
+        if [ ! -d "${opt#*=}" ]; then
```

Comment:
I think it's better to set the case as `--distdir=*` if you want to use `${opt#*=}`.

Reply:
ok

Reply:
you can refer to the case of `--config=*` and put the setting of distdir into `${know_options}` to be consistent.

Reply:
Please note also the `--distdir` settings in `.apollo.bazelrc` generated by `./apollo.sh config`. 
Question: Does bazel support multiple `--distdir` settings ? If not, how to keep consistent across the whole Apollo project ?

And, shouldn't `apollo_test.sh`, `apollo_coverage.sh`  etc. be considered for `--distdir` ?

Reply:
> Please note also the `--distdir` settings in `.apollo.bazelrc` generated by `./apollo.sh config`.
> Question: Does bazel support multiple `--distdir` settings ? If not, how to keep consistent across the whole Apollo project ?
> 
> And, shouldn't `apollo_test.sh`, `apollo_coverage.sh` etc. be considered for `--distdir` ?

A possible solution for consistent `--distdir` settings is setting up an ENV variable (e.g. `APOLLO_BAZEL_DISTDIR`).
All scripts, then use the value of `APOLLO_BAZEL_DISTDIR` as `--distdir` destination.